### ISSUE TITLE
on the website it shows a free plan but there is no free plan

### ISSUE THREAD

**Issue by LukeDevsE** [alamorre/react-chat-engine#146]
on the chat engine website it shows this 
![image](https://user-images.githubusercontent.com/108841495/186550865-403a3b7a-3f49-4104-bb71-49840b07da47.png)
but it forces me to use the light plan

**Comment by JustincHedrick** [comment]
Just ran in to this as well. ChatEngine seems to force you to pay 7 dolars a month now.

**Comment by Alexie7777** [comment]
I'm a new user, seems like the free trials only last for 10 days.

**Comment by itsaalexk** [comment]
I was wondering the same thing, I started this project because they had a free plan but after you create an account it gives you a trial

**Comment by nikoladjokic96** [comment]
Me too, this is sad.

**Comment by TheHumanistX** [comment]
Yup same lol so forget this platform. It literally says I am on the basic plan:
![image](https://user-images.githubusercontent.com/42555733/224428690-6168ba21-3fd0-4de7-8087-d99e32a8be7b.png)

But cannot access the API, clearly from the 429 and 404 messages... oh and it says "Dedicated Support" but no one has answered the support chat and it's been hours 🤣🤣🤣 GREAT way to make people NOT use your product!
